x-pack/plugins/uptime/common/translations.ts

```diff
@@ -32,4 +32,7 @@ export const MonitorStatusTranslations = {
   name: i18n.translate('xpack.uptime.alerts.monitorStatus.clientName', {
     defaultMessage: 'Uptime monitor status',
   }),
+  description: i18n.translate('xpack.uptime.alerts.monitorStatus.description', {
+    defaultMessage: 'Alert when the status of an Uptime monitor is down.',
```

Comment:
i think this need a better copy, uptime monitor status alert also incorporate availability alert 
![image](https://user-images.githubusercontent.com/3505601/97459479-1c644e80-193c-11eb-9f3c-d079227817bb.png)
 
so may be something like `monitor is down or availability crosses set threshold?`

help needed here @paulb-elastic @drewpost

Reply:
Alert when your synthetic monitor is down or its availability breaches an availability threshold

cc @shahzad31

Reply:
Thank you for the feedback! I went ahead and updated the description for this Uptime alert here: https://github.com/elastic/kibana/pull/81850/commits/3c83b6220d193172f6c571f8ef11c101cbbc1905.